code/game/gamemodes/cult/runes.dm

```diff
@@ -478,15 +478,12 @@ structure_check() searches for nearby cultist structures required for the invoca
 	send_to_playing_players('sound/effects/dimensional_rend.ogg')
 	var/turf/T = get_turf(src)
 	sleep(40)
+	SSticker.mode.eldergod = 0
```

Comment:
What do you mean for it's numerical value ? Can this have value other than 1 or 0 right now?